app/org/sagebionetworks/bridge/services/InstantExportViaSqsService.java

```diff
@@ -0,0 +1,74 @@
+package org.sagebionetworks.bridge.services;
+
+import javax.annotation.Nonnull;
+
+import com.amazonaws.services.sqs.AmazonSQSClient;
+import com.amazonaws.services.sqs.model.SendMessageResult;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import org.sagebionetworks.bridge.config.BridgeConfig;
+import org.sagebionetworks.bridge.models.studies.StudyIdentifier;
+
+/**
+ * Implementation of {@link InstantExportService} that connects to the Bridge Exporter Service via SQS.
+ */
+@Component
+public class InstantExportViaSqsService implements InstantExportService{
+    private static final Logger logger = LoggerFactory.getLogger(InstantExportViaSqsService.class);
+
+    private static final ObjectMapper JSON_OBJECT_MAPPER = new ObjectMapper();
+    // constants - these are package scoped so unit tests can access them
+    static final String CONFIG_KEY_EXPORTER_SQS_QUEUE_URL = "exporter.request.sqs.queue.url";
+    static final String REQUEST_KEY_STUDY_WHITE_LIST = "studyWhitelist";
+    static final String REQUEST_KEY_EXPORT_TYPE = "exportType";
+    static final String REQUEST_TAG = "BridgePF Instant Export Request";
```

Comment:
We should add more metadata to this tag. A tag that just says "BridgePF Instant Export Request" doesn't really tell me what the request is.